Content.Server/_NF/GameRule/AdventureRuleComponent.cs

```diff
@@ -5,5 +5,9 @@ namespace Content.Server.GameTicking.Rules.Components;
 [RegisterComponent, Access(typeof(NfAdventureRuleSystem))]
 public sealed partial class AdventureRuleComponent : Component
 {
-
+    public readonly List<EntityUid> NFPlayerMinds = new();
+    public readonly List<EntityUid> CargoDepots = new();
+    public readonly List<EntityUid> MarketStations = new();
+    public readonly List<EntityUid> OptionalPOIs = new();
+    public readonly List<EntityUid> UniquePOIs = new();
```

Comment:
Aren't these unused?

Reply:
they are the appropriate place to be storing these lists, at the moment its being stored in variables on the system side but ultimately it should be stored instead on the rule component itself during runtime here, just hadnt yet gotten to handling that in the system appropriately yet